<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QCheckBox" name="fromStartCheckBox">
       <property name="text">
        <string>Search from &amp;start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="caseCheckBox">
       <property name="text">
        <string>Match &amp;case</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="findButton">
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="moreButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>More</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QWidget" name="extensionWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="wholeCheckBox">
        <property name="text">
         <string>Match &amp;whole words</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="regexCheckBox">
        <property name="text">
         <string>Match &amp;regular expression</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="backwardsCheckBox">
        <property name="text">
         <string>Search &amp;backwards</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>moreButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>extensionWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>FindDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
